### PR TITLE
!!!TASK: Adjust to the new Package API

### PR DESCRIPTION
**Pull request by mficzel**

### CODE REVIEW

**Comment by mficzel** [comment]
This one relies on https://github.com/neos/flow-development-collection/pull/716

**Comment by kitsunet** [comment]
Just for comparison that's what I had locally:
https://github.com/neos/neos-development-collection/pull/1237

**Comment by mficzel** [comment]
I cannot proceed further until https://github.com/neos/neos-development-collection/pull/1237 is resolved.

File: TYPO3.Neos.Kickstarter/Classes/Service/GeneratorService.php [hunk 1]
```diff
@@ -45,11 +42,16 @@ class GeneratorService extends \TYPO3\Kickstart\Service\GeneratorService
      */
     public function generateSitePackage($packageKey, $siteName)
     {
-        $packageMetaData = new MetaData($packageKey);
-        $packageMetaData->addConstraint(new PackageConstraint(MetaDataInterface::CONSTRAINT_TYPE_DEPENDS, 'TYPO3.Neos'));
-        $packageMetaData->addConstraint(new PackageConstraint(MetaDataInterface::CONSTRAINT_TYPE_DEPENDS, 'TYPO3.Neos.NodeTypes'));
-        $packageMetaData->addConstraint(new PackageConstraint(MetaDataInterface::CONSTRAINT_TYPE_SUGGESTS, 'TYPO3.Neos.Seo'));
-        $this->packageManager->createPackage($packageKey, $packageMetaData, null, 'typo3-flow-site');
+        $this->packageManager->createPackage($packageKey, [
+            'type' => 'typo3-flow-site',
```
Comment (johannessteu) [inline TYPO3.Neos.Kickstarter/Classes/Service/GeneratorService.php]: should be `neos-site` with https://github.com/neos/neos-development-collection/pull/1242 in mind

Comment (mficzel) [inline TYPO3.Neos.Kickstarter/Classes/Service/GeneratorService.php]: Is altered ... please retest

**Comment by mficzel** [comment]
The mentioned points are fixed .. i think this is ready for a last review

**Comment by gerhard-boden** [comment]
works now, thanks!

**Pull request merged**